cluster/images/etcd/migrate-if-needed.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
I think we can and should unit test this, by starting etcd2/3, writing something, stopping it, run the script, then start etcd3/2 and verify the written thing is there.